tools/encoder/src/TTFFontEncoder.cpp

```diff
@@ -3,6 +3,8 @@
 #include "GPBFile.h"
 #include "StringUtil.h"
 
+#include <vector>
```

Comment:
There is no real reason to use a vector. Especially when you don't need to dynamically grow on multiple iterations.
Just create the array for the size you need and then access as needed.